.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -160,6 +165,12 @@ runs:
             - operator: Exists
       EOF
 
+      if [[ $ENABLE_LOCAL_ZONES == "true" ]]; then
+        while IFS= read -r zone; do
```

Comment:
Is there a less indirect way for us to handle this? Can we read the items into an array object and then iterate through it? I see what you are doing here, but you can look down at the private cluster VPC endpoint logic to see how this is handled there with subnets and security groups like on L187

Reply:
I'll preface this with I'm not a bash expert, the way I know how to do it though requires bash 4+ IIRC so wouldn't be very portable. I can take a look and see if there's a simpler way to iterate over the output here.